applications/FluidDynamicsApplication/tests/sod_shock_tube_test.py

```diff
@@ -36,8 +36,8 @@ def testSodShockTubeExplicitOSSShockCapturing(self):
     def setUp(self):
         self.print_output = False
         self.print_reference_values = False
-        self.check_absolute_tolerance = 1.0e-8
```

Comment:
to me this tolerance does not appear to be that strict. Are u sure we don't have any pending problem with parallelism?